source/lex.tex

```diff
@@ -296,15 +315,135 @@
+\tcode{\textbackslash U} \grammarterm{hex-quad} \grammarterm{hex-quad}
 designates the character in the translation character set
 whose UCS scalar value is the hexadecimal number represented by
 the sequence of \grammarterm{hexadecimal-digit}s
 in the \grammarterm{universal-character-name}.
 The program is ill-formed if that number is not a UCS scalar value.
+
+\pnum
+A \grammarterm{universal-character-name}
+that is a \grammarterm{named-universal-character}
+designates the character named by its \grammarterm{n-char-sequence}.
+A character is so named if the \grammarterm{n-char-sequence} is equal to
+\begin{itemize}
+\item
+the associated character name or associated character name alias
+specified in ISO/IEC 10646 subclause ``Code charts and lists of character names''
+or
+\item
+the control code alias given in \tref{lex.charset.ucn}.
+\begin{note}
+The aliases in \tref{lex.charset.ucn} are provided for control characters
+which otherwise have no associated character name or character name alias.
+These names are derived from
+the Unicode Character Database's \tcode{NameAliases.txt}.
+For historical reasons, control characters are formally unnamed.
+\end{note}
+\end{itemize}
+\begin{note}
+None of the associated character names,
+associated character name aliases, or
+control code aliases
+have leading or trailing spaces.
+\end{note}
+
+\begin{multicolfloattable}{Control code aliases}{lex.charset.ucn}{ll}
+\unicode{0000}{null} \\
+\unicode{0001}{start of heading} \\
+\unicode{0002}{start of text} \\
+\unicode{0003}{end of text} \\
+\unicode{0004}{end of transmission} \\
+\unicode{0005}{enquiry} \\
+\unicode{0006}{acknowledge} \\
+\unicode{0007}{alert} \\
+\unicode{0008}{backspace} \\
+\unicode{0009}{character tabulation} \\
+\unicode{0009}{horizontal tabulation} \\
+\unicode{000a}{line feed} \\
+\unicode{000a}{new line} \\
+\unicode{000a}{end of line} \\
+\unicode{000b}{line tabulation} \\
+\unicode{000b}{vertical tabulation} \\
+\unicode{000c}{form feed} \\
+\unicode{000d}{carriage return} \\
+\unicode{000e}{shift out} \\
+\unicode{000e}{locking-shift one} \\
+\unicode{000f}{shift in} \\
+\unicode{000f}{locking-shift zero} \\
+\unicode{0010}{data link escape} \\
+\unicode{0011}{device control one} \\
+\unicode{0012}{device control two} \\
+\unicode{0013}{device control three} \\
+\unicode{0014}{device control four} \\
+\unicode{0015}{negative acknowledge} \\
+\unicode{0016}{synchronous idle} \\
+\unicode{0017}{end of transmission block} \\
+\unicode{0018}{cancel} \\
+\unicode{0019}{end of medium} \\
+\unicode{001a}{substitute} \\
+\unicode{001b}{escape} \\
+\unicode{001c}{information separator four} \\
+\unicode{001c}{file separator} \\
+\unicode{001d}{information separator three} \\
+\unicode{001d}{group separator} \\
+\unicode{001e}{information separator two} \\
+\unicode{001e}{record separator} \\
+\unicode{001f}{information separator one} \\
+\unicode{001f}{unit separator} \\
+\columnbreak
+\unicode{0082}{break permitted here} \\
```

Comment:
U+007F | DELETE
-- | --


is missing between these two.

Reply:
Fixed.